Source/WebKit/UIProcess/API/glib/WebKitError.cpp

```diff
@@ -112,6 +112,18 @@ GQuark webkit_snapshot_error_quark()
     return g_quark_from_static_string("WebKitSnapshotError");
 }
 
+/**
+ * webkit_web_extension_match_pattern_error_quark:
+ *
+ * Gets the quark for the domain of Web Extension Match Pattern errors.
+ *
+ * Returns: web extension match pattern error domain.
+ */
```

Comment:
```suggestion
 * Returns: web extension match pattern error domain.
 *
 * Since: 2.48
 */
```

Please double check the rest of this pull request for other missing Since tags.